tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerBuilder.java

```diff
@@ -296,6 +305,13 @@ public JaegerTracerBuilder config(Config config) {
 
         config.get("global").asBoolean().ifPresent(this::registerGlobal);
 
+        config.get("span-processor-type").asString()
+                .ifPresent(it -> spanProcessorType(SpanProcessorType.valueOf(it.toUpperCase())));
+        config.get("exporter-timeout").asLong().ifPresent(it -> exporterTimeout(Duration.ofMillis(it)));
+        config.get("schedule-delay").asInt().ifPresent(it -> scheduleDelay(Duration.ofMillis(it)));
```

Comment:
And once again - use `Duration` here as well.

Reply:
ok

Reply:
Fixed for 3.x